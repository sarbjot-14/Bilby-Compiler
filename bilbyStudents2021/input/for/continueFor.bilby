main{

	
	for(i in <2..7>) {
		
		if(i == 5){
			print "dont print 25" $n;
			continue;
		}
		print i*i $s;

	}
	print $n;

}

