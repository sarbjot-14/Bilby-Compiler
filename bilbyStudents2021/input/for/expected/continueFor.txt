4 9 16 dont print 25
36 49 
